README.md

```diff
@@ -1,2 +1,351 @@
-# year-cohort-projectName (eg: 22-jul-projectName)
-project template repo
+# Recipes API
+
+## API documentation
+
+### Return/Create user userId - story-2
+
+- **URL**
+
+  /users
+
+- **Method:**
+
+  `POST`
+
+- **URL Params**
+
+  none
+
+  **Example:**
+
+  `/users`
+
+## get user
+
+- **Success Response:**
+  - **Code:** 200
+  - **Content:**
+
+```json
+{
+  "message": "Successfully retrieved user",
+  "data": {
+    "userId": "1"
+  }
+}
+```
+
+- **Error Response:**
```

Comment:
What if the 'email address' they gave you isn't an email address? Are you doing any validation responses?

Reply:
done

Reply:
400 is not a server error response

Reply:
you also didn't replicate this below for the 'add if user not existing' route.

attention to details 😩